examples/src/test/java/com/opengamma/strata/examples/marketdata/ExampleMarketDataBuilderTest.java

```diff
@@ -242,14 +243,14 @@ private void assertBuilder(ExampleMarketDataBuilder builder) {
     assertEquals(MARKET_DATA_DATE, snapshot.getValuationDate().getSingleValue());
 
     for (ObservableId id : TIME_SERIES) {
-      assertTrue(snapshot.containsTimeSeries(id), "Time-series not found: " + id);
+      assertFalse(snapshot.getTimeSeries(id).isEmpty(), "Time-series not found: " + id);
```

Comment:
This isn't the same logic. Not sure it matters, but cannot distinguish between time series that is present but empty and one that is absent.